File: lib/layer2_core/light/e_surface_bright_light.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/light/system.dart';
import '../../layer1_raw/light/standard.dart';
import '../../layer1_raw/light/blue.dart';
import '../../layer1_raw/light/green.dart';
import '../../layer1_raw/light/red.dart';
import '../../layer1_raw/light/nothing.dart';

class ESurfaceBrightLight {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyl9;
      case 'standard': return cstl9;
      case 'blue': return cbl9;
      case 'green': return cgl9;
      case 'red': return crl9;
      case 'nothing': return cnol9;
      default: return cstl9;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/light/ESurfaceBrightLight.kt
package com.example.design_engine.layer2_core.light
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object ESurfaceBrightLight {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyl9
            AppTheme.STANDARD -> R.color.cstl9
            AppTheme.BLUE -> R.color.cbl9
            AppTheme.GREEN -> R.color.cgl9
            AppTheme.RED -> R.color.crl9
            AppTheme.NOTHING -> R.color.cnol9
        }
    }
}
